components/cdc/src/endpoint.rs

```diff
@@ -350,14 +349,103 @@ impl ResolvedRegionHeap {
     fn is_empty(&self) -> bool {
         self.heap.is_empty()
     }
+}
 
-    fn clear(&mut self) {
-        self.heap.clear();
-    }
+#[derive(Default, Debug)]
+pub(crate) struct Advance {
+    // multiplexing means one region can be subscribed multiple times in one `Conn`,
+    // in which case progresses are grouped by (ConnId, request_id).
+    pub(crate) multiplexing: HashMap<(ConnId, u64), ResolvedRegionHeap>,
```

Comment:
can we declare a new data type `RequestID` to replace the `u64`

Reply:
Many lines will be changed by this replacing. I prefer to do it in another PR.